src/Graph.ts

```diff
@@ -63,7 +62,7 @@ export default class Graph {
 	isPureExternalModule: (id: string) => boolean;
 	legacy: boolean;
 	load: (id: string) => Promise<SourceDescription | string | void>;
-	handleMissingExport: MissingExportHook;
+	handleMissingExport: (module: Module, name: string, otherModule: string, start?: number) => void;
```

Comment:
I take it `module` is the importer here, and start is the index in the importer? How about `name, importingModule, importedModule, importerStart`?

Reply:
updated the wording and order.